src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -613,12 +613,9 @@ function updateMatchingLineNumbers(): void {
   isProcessingInternally.value = true
   regExpError.value = null
 
-  // Avoids searching when one can expect a very large number of results. The numbers here are determined purely by gut feel and not by any scientific reasoning. Feel free to tweak them.
-  const isSmallEnoughForCostlySearch = query.value.length >= 3 || props.code.length < 1000
-  const shouldPerformSearch = query.value.length > 0 && (isRegExpMode.value || (!isRegExpMode.value && isSmallEnoughForCostlySearch))
   let totalMatchingLineNumbers: number[] = []
 
-  if (shouldPerformSearch) {
+  if (query.value.length > 0) {
```

Comment:
question: why would it ever make sense to query based on 1 character?

Also curious why you're removing the "expensive" restriction; I remember you wanting it in place originally

Reply:
It comes up occasionally when we do our test Friday sessions that people wonder why they can’t search for "certain numbers".

Originally I had this in place because we had some (what turned out to be unrelated) performance issues while searching. Those were caused by syntax highlighting being very slow for some obscure reasons. We don’t have those issues anymore and it should be fine to search longer content.

Reply:
The search is still debounced, correct?

Reply:
Yes, at 150 ms.